update-resources.sh

```diff
@@ -210,6 +210,22 @@ function add_chisel() {
   wget -qO- "https://github.com/jpillora/chisel/releases/latest/download/chisel_${version}_linux_386.gz" | gzip -d > ./linux/chisel/chisel32
 }
 
+function add_ligolo-ng() {
+  info "Downloading ligolo-ng agents"
+  chkfs "./windows/ligolo-ng/"
+  chkfs "./linux/ligolo-ng/"
+  wget -qO- "https://github.com/nicocha30/ligolo-ng/releases/download/v0.4.3/ligolo-ng_agent_0.4.3_Linux_64bit.tar.gz"|tar -xzvf - -C ./linux/ligolo-ng
+  mv ./linux/ligolo-ng/agent ./linux/ligolo-ng/agent64
+  rm -rf ./linux/ligolo-ng/README.md ./linux/ligolo-ng/LICENSE
+  wget -qO- "https://github.com/nicocha30/ligolo-ng/releases/download/v0.4.3/ligolo-ng_agent_0.4.3_Linux_ARM64.tar.gz"|tar -xzvf - -C ./linux/ligolo-ng
+  mv ./linux/ligolo-ng/agent ./linux/ligolo-ng/agent64-arm
+  rm -rf ./linux/ligolo-ng/README.md ./linux/ligolo-ng/LICENSE 
+  wget -q -O ./windows/ligolo-ng/agent.zip "https://github.com/nicocha30/ligolo-ng/releases/download/v0.4.3/ligolo-ng_agent_0.4.3_Windows_64bit.zip"
+  unzip -q -o -d ./windows/ligolo-ng/agent64.exe ./windows/ligolo-ng/agent.zip
+  mv ./windows/ligolo-ng/agent64.exe/agent.exe ./windows/ligolo-ng/agent.exe
+  rm -rf ./windows/ligolo-ng/agent64.exe/ ./windows/ligolo-ng/agent.zip
```

Comment:
Can you synchronize with latest changes from main, and remove the quiet options from `wget` and `unzip` commands, as well as add verbosity to the `rm` and `m`v commands ?